storm-client/src/jvm/org/apache/storm/executor/Executor.java

```diff
@@ -680,4 +677,8 @@ public HashMap getSharedExecutorData() {
     public void setLocalExecutorTransfer(ExecutorTransfer executorTransfer) {
         this.executorTransfer = executorTransfer;
     }
+
+    public void incrementErrorCount() {
```

Comment:
maybe change this to incrementReportedErrorCount so it is specific to reporter errors?

Reply:
updated